### PR TITLE
Remove unused course structure tables and jobs

### PR DESCRIPTION
**Pull request by bmedx**
Analytics Pipeline Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] If you have a migration please contact data engineering team before merging.
  - [ ] Before merging run full acceptance tests suite and provide URL for the acceptance tests run.
  - [ ] A member of data engineering team has approved the pull request.

### CODE REVIEW

File: edx/analytics/tasks/tests/acceptance/test_internal_reporting_course_structure.py [hunk 1]
```diff
@@ -30,22 +30,6 @@ def setUp(self):
                 url_path_join(self.warehouse_path, 'course_block_raw', 'dt=' + self.DATE, file_name)
             )
 
-    @when_vertica_available
-    def test_internal_reporting_course_structure(self):
```
Comment (brianhw) [inline edx/analytics/tasks/tests/acceptance/test_internal_reporting_course_structure.py:34]: This is the only test here.   I assume that you can remove the entire file, not just the test.

File: edx/analytics/tasks/warehouse/load_internal_reporting_course_structure.py [hunk 2]
```diff
@@ -642,42 +642,6 @@ def on_success(self):  # pragma: no cover
...
-        return self.required_tasks
-
-    @property
-    def partition(self):
-        """The table is partitioned by date."""
-        return HivePartition('dt', self.date.isoformat())  # pylint: disable=no-member
-
-    @property
-    def insert_source_task(self):
-        return AllCourseBlockRecordsTask(
```
Comment (brianhw) [inline edx/analytics/tasks/warehouse/load_internal_reporting_course_structure.py:662]: If you remove this task, then you remove the task that generates the table.  The LoadInternalReportingCourseStructureToWarehouse task below uses an ExternalURL to point to S3 data, but nothing updates it.  And that might be fine.   But then you no longer need any of the classes above this.

File: setup.cfg [hunk 3]
```diff
@@ -69,7 +69,6 @@ edx.analytics.tasks =
     load-google-sheet-snowflake = edx.analytics.tasks.warehouse.load_google_sheet_to_warehouse:LoadGoogleSpreadsheetsToSnowflakeWorkflow
     load-insights= edx.analytics.tasks.warehouse.load_warehouse_insights:LoadInsightsTableToVertica
     load-internal-active-users = edx.analytics.tasks.warehouse.load_internal_reporting_active_users:LoadInternalReportingActiveUsersToWarehouse
-    load-internal-course-structure = edx.analytics.tasks.warehouse.load_internal_reporting_course_structure:LoadCourseBlockRecordToVertica
```
Comment (brianhw) [inline setup.cfg:72]: If you want load_internal_reporting_course_structure to be loaded as a task plugin, then I think you need an entry in here that loads one of the remaining task classes.  E.g. if LoadInternalReportingCourseStructureToWarehouse is remaining to be called from load-warehouse.

**Review by HassanJaveed84 — APPROVED** [review]
LGTM

**Review by brianhw — APPROVED** [review]
LGTM.  👍 

(Though I guess I thought we had gotten rid of the BigQuery code....)

**Pull request merged**